### ISSUE TITLE
SalesAnalyst

### ISSUE THREAD

**Issue by dphilla** [mikeyduece/black_thursday#3]

**Comment by dphilla** [comment]
Makes Sales Analyst lib/test files, with working dependencies

**Comment by mikeyduece** [comment]
Need to maybe look at putting methods in SE to call all of each invoice, merchant, item, etc. Include Stats module in SA or maybe put each analyst in its own file and call them in SA either that or SA not a module and will hold all of our calculations

**Comment by mikeyduece** [comment]
can maybe refactor - taking out the instantiations in SA. if we call SE in the initialization, we should be able to call those functions in SA.

**Comment by mikeyduece** [comment]
oh and I could use a walkthrough of the analytical methods you wrote :D looks good bro, that woulda taken me forever:P

**Comment by dphilla** [comment]
yeah you're exactly right! thats what I overlooked. I re-read the spec and realized I should have added those methods in the SE. I have everything solid now with the tests but I'm gonna go ahead and refactor those parts into the SE before I throw you a PR to Merge.
